.travis.yml

```diff
@@ -169,11 +169,14 @@ matrix:
 before_script:
   # Bootstrap everything (including letor, which isn't done
   # by default), then configure using our chosen compiler.
+  - eval "$(gimme 1.14)"
+  - eval env
```

Comment:
What's the purpose of `eval env` here?

Reply:
I guess you're just wanting to show the currently set environment variables here?  If so, just `env` will do that (I don't think you're never going to want to use eval to run a fixed command). 

But this is really debugging code so shouldn't be added - there's a length limit on travis logs, so printing out everything you might ever want to know isn't a workable approach.

Reply:
yea..I have used it for debugging ..ill be more careful to remove all debugging statements from now on before merging to my master branch.. :-)